main.go

```diff
@@ -129,6 +140,28 @@ func sendMetrics(event *corev2.Event) error {
 		return err
 	}
 
+	// Add the check status field as a metric if requested. Measurement recorded as the check name.
+	if config.CheckStatusMetric {
```

Comment:
We probably want to add a check here to ensure that the event actually contains a check (for example, statsd metrics don't!), otherwise `event.Check.Name` and `event.Check.Status` could panic. We should return or log an error if the following `if` block fails. Might make sense to log it so the same handler config can be reused for different cases.

```suggestion
	if config.CheckStatusMetric && event.HasCheck() {
```